bsc-plugin/src/plugin.spec.ts

```diff
@@ -2013,6 +2013,37 @@ describe('RooibosPlugin', () => {
         });
     });
 
+    describe('does not break class extending', () => {
+        it('does not prevent valid scope based diagnostics for node tests', () => {
```

Comment:
@chrisdp I think describe and test strings need updated to represent what this test is actually doing.

Reply:
I thin the test string is, describe could use an update for sure.